pkgs/tools/X11/sbs/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchzip, libX11, imlib2, libXinerama, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "sbs";
+  version = "1.0.0";
+
+  src = fetchzip {
+    url = "https://github.com/onur-ozkan/${pname}/archive/refs/tags/v${version}.tar.gz";
+    sha256 = "sha256-Zgu9W/3LwHF/fyaPlxmV/2LdxilO1tU0JY/esLnJVGY=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ imlib2 libX11 libXinerama ];
+
+  buildPhase = ''
+    mkdir -p $out/bin
+    make ${pname}
+    mv ${pname} $out/bin/.
+  '';
```

Comment:
```suggestion
```
Looking at the Makefile, this should not be needed. 
The build phase calls `make` per default (i.e. `make all`?). 
Creating `$out` and installing the binary should be done in the `InstallPhase`, but that happens automatically, since the default is to call `make install`.

Reply:
> Creating $out and installing the binary should be done in the InstallPhase, but that happens automatically, since the default is to call make install.

Without the following block:

```nix
  installPhase = ''
    mkdir -p $out/bin
    mv ${pname} $out/bin/.
  '';
```

I am not able to see the `sbs` binary under `result/bin`, should I remove it anyway?

Reply:
As an alternative I would recommend to use `makeFlags = [ "PREFIX=$(out)" ];`.
This leaves the original `installPhase` intact.

Reply:
Yeah that works, changed it. Sorry, still learning how the packaging works for Nix :)